plugin.php

```diff
@@ -35,7 +35,7 @@ function domainlimit_link_filter( $original_return, $url, $keyword = '', $title
 	// The plugin hook gives us the raw URL input by the user, but
 	// it needs some cleanup before it's suitable for parse_url().
 	$url = yourls_encodeURI( $url );
-	$url = yourls_escape( yourls_sanitize_url( $url) );
+	// $url = yourls_escape( yourls_sanitize_url( $url) );
```

Comment:
`yourls_escape()` was probably never the right function to use here. Rather than commenting it out, just remove it.

```suggestion
	$url = yourls_sanitize_url( $url );
```